index.js

```diff
@@ -47,6 +47,7 @@ instance.prototype.init = function(is_config) {
 		self.config.client_secret,
 		self.config.client_redirect_url,
 		scopes,
+		parseInt(self.config.fetch_max_count) || 10,
```

Comment:
`parseInt` isn't necessary for a number field type.

Reply:
Fixed